From 4eb0946caf764350e0787fc6fbefe260ed949873 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Uwe=20Kleine-K=C3=B6nig=20=28The=20Capable=20Hub=29?=
 <u.kleine-koenig@baylibre.com>
Date: Fri, 17 Jul 2026 16:50:47 +0200
Subject: [PATCH 09282/15333] misc: Drop unused assignment of spi_device_id
 driver data
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The drivers don't make use of the values that were explicitly assigned
to the .driver_data members. Drop these assignments.

Signed-off-by: Uwe Kleine-König (The Capable Hub) <u.kleine-koenig@baylibre.com>
Link: https://patch.msgid.link/e2ad3f93b358cc246a64a3d0a7c3f037b2201412.1784299069.git.u.kleine-koenig@baylibre.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/misc/gehc-achc.c           | 4 ++--
 drivers/misc/lattice-ecp3-config.c | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/misc/gehc-achc.c b/drivers/misc/gehc-achc.c
index b8fca4d393c6..1abfa6c257de 100644
--- a/drivers/misc/gehc-achc.c
+++ b/drivers/misc/gehc-achc.c
@@ -538,8 +538,8 @@ static int gehc_achc_probe(struct spi_device *spi)
 }
 
 static const struct spi_device_id gehc_achc_id[] = {
-	{ "ge,achc", 0 },
-	{ "achc", 0 },
+	{ "ge,achc" },
+	{ "achc" },
 	{ }
 };
 MODULE_DEVICE_TABLE(spi, gehc_achc_id);
diff --git a/drivers/misc/lattice-ecp3-config.c b/drivers/misc/lattice-ecp3-config.c
index 93949df3bcff..2b93d5660bf2 100644
--- a/drivers/misc/lattice-ecp3-config.c
+++ b/drivers/misc/lattice-ecp3-config.c
@@ -219,8 +219,8 @@ static void lattice_ecp3_remove(struct spi_device *spi)
 }
 
 static const struct spi_device_id lattice_ecp3_id[] = {
-	{ "ecp3-17", 0 },
-	{ "ecp3-35", 0 },
+	{ "ecp3-17" },
+	{ "ecp3-35" },
 	{ }
 };
 MODULE_DEVICE_TABLE(spi, lattice_ecp3_id);
-- 
2.55.0

